src/components/Modals/SubjectLockedModal/SubjectLockedModal.js

```diff
@@ -19,7 +19,7 @@ export default function SubjectLockedModal({ onBack }) {
     >
       <Text size='large'>Subject locked</Text>
       <StyledText>
-        This subject cannot be accessed because <b>Erin Green</b> is currently accessing it.
+        This subject cannot be accessed because <b>{lockedBy}</b> is currently accessing it.
```

Comment:
I think [`<strong>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong) is a better element to use rather than `<b>`. The `<b>` element is one of those older HTML 4 elements that is discouraged now to use because it's not semantic. The other option would be to wrap this in a span and use CSS to style it.